src/military/military.cpp

```diff
@@ -5181,7 +5189,7 @@ void update_land_battles(sys::state& state) {
 					case unit_type::special:
 						state.world.land_battle_get_defender_support_lost(b) += str_damage;
 						break;
-					default:
```

Comment:
why did you remove the default case? yes, it shouldn't occur in a sane world, but if something really wacky has happened (like ship types appearing here), that is what it exists for. I also makes the static analysis tools happy

Reply:
i think i removed it by accident